### PR TITLE
Vof examples improvement

### LINKED ISSUE #580: Improve VOF documentation
The VOF example for Rayleigh Taylor and breaking dam are not set-up optimally (they don't use adaptive initial refinement) and there are some small missing elements in the documentations. This should be fixed.

### PR DESCRIPTION
**Pull request by blaisb**
# Description of the problem

- The VOF examples were slightly incomplete and were not using the new initial refinement approach to set up the field
-  The hyphenate change broke my other PR, this one is a newer version of this

# Description of the solution

- Fix initial condition
- Rerun all examples to make sure everything is still good
- Clean python code to actually extract the time from the pvd file
- Indent prm files
- Add white background to the rising bubble initial condition
- Remove the filtering of the position to extract the raw value. It's actually closer now...

This PR closes #580

### CODE REVIEW

**Review by lpsaavedra — COMMENTED** [review]
Oh.. I just noticed that the equations in the dam-break example and the rising bubble one are in very weird boxes...  can I correct them?

**Comment by blaisb** [comment]
> Oh.. I just noticed that the equations in the dam-break example and the rising bubble one are in very weird boxes... can I correct them?

Absolutely, please feel free to do so.

File: doc/source/examples/multiphysics/dam-break/dam-break.rst [hunk 1]
```diff
@@ -10,16 +10,16 @@ This example simulates the dam break experiments of `Martin and Moyce (1952)`_.
...
 - Unsteady problem handled by an adaptive BDF1 time-stepping scheme 
 - The use of a python script for post-processing data
 
 
-------------------------
-Location of the example
-------------------------
-``examples/multiphysics/dam-break-VOF/gls-VOF-dam-break-Martin-and-Moyce.prm``
+--------------------------------------------
+Location of the files used in this example
```
[the remark this answers is not in the document]
Comment (blaisb) [reply doc/source/examples/multiphysics/dam-break/dam-break.rst:20]: That was Amishga's conclusion regarding the fact that examples at multiple files and etc., so I would rather use move towards that. If you have time to help, i'll be glad to review that :).
I'll fix the Rayleigh-Taylor :).

[the remark this answers is not in the document]
Comment (lpsaavedra) [reply doc/source/examples/multiphysics/dam-break/dam-break.rst:20]: Sure, I will do it later today. I fixed the equations and the Rayleigh-Taylor title already

File: doc/source/examples/multiphysics/rayleigh-taylor-instability/rayleigh-taylor-instability.rst [hunk 2]
```diff
@@ -169,8 +188,9 @@ The ``initial refinement`` of the mesh is equal to 7, but we use a mesh adaptati
       set max refinement level    	= 7
       set min refinement level    	= 5
       set frequency               	= 1
-      set fraction refinement     	= 0.9
-      set fraction coarsening     	= 0.001
+      set fraction refinement     	= 0.99
+      set fraction coarsening     	= 0.01
+      set initial refinement steps  = 4
```
Comment (hepap) [inline doc/source/examples/multiphysics/rayleigh-taylor-instability/rayleigh-taylor-instability.rst:193]:
```suggestion
      set fraction coarsening     	= 0.01
      set initial refinement steps  	= 4
```

File: doc/source/examples/multiphysics/rayleigh-taylor-instability/rayleigh-taylor-instability.rst [hunk 3]
```diff
@@ -18,12 +18,12 @@ Features
...
-Location of the example
-------------------------
-``examples/multiphysics/rayleigh-taylor-instability/rayleigh-taylor-instability-constant-refinement.prm``
+--------------------------------------------
+Location of the files used in this example
+--------------------------------------------
+``examples/multiphysics/rayleigh-taylor-instability/rayleigh-taylor-instability_constant_sharpening.prm``
 
-``examples/multiphysics/rayleigh-taylor-instability/rayleigh-taylor-instability-adaptive-refinement.prm``
+``examples/multiphysics/rayleigh-taylor-instability/rayleigh-taylor-instability_adaptive_sharpening.prm``
```
Comment (hepap) [inline doc/source/examples/multiphysics/rayleigh-taylor-instability/rayleigh-taylor-instability.rst:26]: I don't remember if we want - or _ in the file name, but either way, the name of the file in the .rst has to match with the file in the example folder. If it is _ (e.g., rayleigh-taylor-instability_constant_sharpening.prm), we have to rename the .prm :) 

We can also reproduce what is done in the order example and just refer to the example folder:
```suggestion
``examples/multiphysics/rayleigh-taylor-instability/``
```

File: doc/source/examples/multiphysics/rising-bubble/rising-bubble.rst [hunk 4]
```diff
@@ -143,29 +137,29 @@ The interface sharpening method and its parameters are explained in :doc:`../dam
...
 	  end
 	  subsection surface tension force
-	    set enable      = true
+	    set enable                          = true
 	    set surface tension coefficient 	= 24.5
 	    set phase fraction gradient filter 	= 0.0005
-	    set curvature filter		= 0.0005
-	    set output auxiliary fields 	= true
+	    set curvature filter		        = 0.0005
+	    set output auxiliary fields 	    = true
```
Comment (hepap) [inline doc/source/examples/multiphysics/rising-bubble/rising-bubble.rst:150]: The indentation is off, but i can't fix it with a suggestion.

File: doc/source/examples/multiphysics/rising-bubble/rising-bubble.rst [hunk 5]
```diff
@@ -143,29 +137,29 @@ The interface sharpening method and its parameters are explained in :doc:`../dam
 	#---------------------------------------------------
 	subsection VOF
 	  subsection interface sharpening
-	    set enable      = true
-	    set threshold   = 0.5
+	    set enable              = true
+	    set threshold           = 0.5
 	    set interface sharpness	= 1.4
```
Comment (hepap) [inline doc/source/examples/multiphysics/rising-bubble/rising-bubble.rst:142]: The indentation is off, but i can't fix it with a suggestion.

File: doc/source/examples/multiphysics/rising-bubble/rising-bubble.rst [hunk 6]
```diff
@@ -258,15 +253,15 @@ Running the simulation
 
 Call the gls_navier_stokes_2d by invoking:  
 
-``mpirun -np 8 gls_navier_stokes_2d rising-bubble-VOF.prm``
+``mpirun -np 8 gls_navier_stokes_2d rising-bubble_VOF.prm``
```
Comment (hepap) [inline doc/source/examples/multiphysics/rising-bubble/rising-bubble.rst:256]:
```suggestion
``mpirun -np 8 gls_navier_stokes_2d rising-bubble.prm``
```

File: doc/source/parameters/cfd/mesh_adaptation_control.rst [hunk 7]
```diff
@@ -76,3 +76,5 @@ This subsection controls the mesh adaptation method, with default values given b
 .. warning::
 	The ``max number elements`` parameter puts a hard limit on the number of cells in the domain, even if the ``fraction refinement`` is increased.
 
+* The number of initial (before solving) adaptive refinement steps is controlled by the ``initial refinement steps`` parameter. With an ``initial refinement steps`` larger than 0, the triangulation is refined adaptively before the solver starts solving the problem. This enables the user to adapt the initial mesh to the initial condition. For example, if the simulation is a VOF simulation, it is ideal to have an initial mesh that captures the interface between the fluids accurately. This is achieved by refining the mesh using the dynamic mesh adaptation parameters and reapplying the initial condition after each adaptation. This process will be repeated ``initial refinement steps`` times.
+
```
Comment (hepap) [inline doc/source/parameters/cfd/mesh_adaptation_control.rst:80]: I understand now :)

File: examples/multiphysics/dam-break/dam-break-2d.py [hunk 8]
```diff
@@ -1,6 +1,6 @@
 #############################################################################
 """
-Postprocessing code for gls_VOF_dam-break_Martin_and_Moyce example
+Postprocessing code for gls_dam-break_Martin_and_Moyce example
```
Comment (hepap) [inline examples/multiphysics/dam-break/dam-break-2d.py:3]:
```suggestion
Postprocessing code for dam-break_Martin_and_Moyce example
```

File: examples/multiphysics/rayleigh-taylor-instability/rayleigh-taylor-instability-adaptive-sharpening.prm [hunk 9]
```diff
@@ -0,0 +1,174 @@
...
+# Simulation Control
+#---------------------------------------------------
+
+subsection simulation control
+  set method         = bdf1
+  set time end       = 0.75
+  set time step      = 0.0002
+  set adapt          = true
+  set max cfl        = 0.9
+  set output name    = rayleigh-taylor_adaptive_sharpening
```
Comment (hepap) [inline examples/multiphysics/rayleigh-taylor-instability/rayleigh-taylor-instability-adaptive-sharpening.prm:13]:
```suggestion
  set output name    = rayleigh-taylor-adaptive-sharpening
```

File: examples/multiphysics/rayleigh-taylor-instability/rayleigh-taylor-instability-constant-sharpening.prm [hunk 10]
```diff
@@ -0,0 +1,172 @@
...
+# Simulation Control
+#---------------------------------------------------
+
+subsection simulation control
+  set method         = bdf1
+  set time end       = 0.75
+  set time step      = 0.0002
+  set adapt          = true
+  set max cfl        = 0.9
+  set output name    = rayleigh-taylor_constant_sharpening
```
Comment (hepap) [inline examples/multiphysics/rayleigh-taylor-instability/rayleigh-taylor-instability-constant-sharpening.prm:13]:
```suggestion
  set output name    = rayleigh-taylor-constant-sharpening
```

**Pull request merged**